webapp/src/pages/Settings.tsx

```diff
@@ -63,7 +70,7 @@ type SubConfigKeys = keyof PickByValue<AzimuthConfig, object | null>;
 
 const CUSTOM_METRICS: string[] = ["Accuracy", "Precision", "Recall", "F1"];
 const ADDITIONAL_KWARGS_CUSTOM_METRICS = ["Precision", "Recall", "F1"];
-
+const SUPPORTED_LANGUAGES = ["en", "fr"];
```

Comment:
I think it would be wise to enforce de type here, so we notice if the supported languages change.
```suggestion
const SUPPORTED_LANGUAGES: SupportedLanguage[] = ["en", "fr"];
```